src/lib.rs

```diff
@@ -274,7 +274,7 @@ fn get_runner(serve: bool) -> Result<PathAndArgs> {
             (
                 PathAndArgs::new("wasmtime")
                     .args(if serve {
-                        vec!["serve"]
+                        vec!["serve", "-S", "common", "-S", "http"]
```

Comment:
I think "common" is being deprecated. Too early to use "cli"?

Reply:
Yeah, good catch. I have an issue to track making that change: https://github.com/bytecodealliance/cargo-component/issues/306

It was recently failing CI tests switching to "cli" instead of "common". Need to track down why it is still grabbing an older release of wasmtime.